src/components/Filter/Filter.jsx

```diff
@@ -0,0 +1,98 @@
+              Price (High to Low)</label>
+          </div>
+        </div>
+        <div className="filter-group">
+          <p className="text-m"><b>Metal</b></p>
+          <div className="filter-item">
+            <label>
+              <input type="checkbox" name="metal" id="filter-metal" 
+              onChange={()=>productDispatch({type:"GOLD"})}/>
+              Gold</label>
+          </div>
+          <div className="filter-item">
+            <label>
+              <input type="checkbox" name="metal" id="filter-metal" 
+              onChange={()=>productDispatch({type:"SILVER"})}/>
+              Silver</label>
+          </div>
+          <div className="filter-item">
+            <label>
+              <input type="checkbox" name="metal" id="filter-metal" onChange={()=>productDispatch({type:"PLATINUM"})}/>
+              Platinum</label>
+          </div>
+        </div>
+        <div className="filter-group">
+          <p className="text-m"><b>Category</b></p>
+          <div className="filter-item">
+            <input type="checkbox" name="Category" id="filter-category" onChange={()=>productDispatch({type:"ring"})}/>
+            <label>Ring</label>
+          </div>
+          <div className="filter-item">
+            <input type="checkbox" name="Category" id="filter-category" onChange={()=>productDispatch({type:"earring"})}/>
+            <label>Earring</label>
+          </div>
+          <div className="filter-item">
+            <input type="checkbox" name="Category" id="filter-category" onChange={()=>productDispatch({type:"necklace"})}/>
+            <label>Necklace</label>
+          </div>
+          <div className="filter-item">
+            <input type="checkbox" name="Category" id="filter-category" onChange={()=>productDispatch({type:"bangles"})}/>
+            <label>Bangles</label>
+          </div>
+        </div>
+        <div className="filter-group">
+          <p className="text-m"><b>Rating</b></p>
+          <div className="filter-item">
+            <input type="radio" name="rating" id="filter-rating" onChange={()=>productDispatch({type:"RATING",payload:4})}/>
+            <label for="">4 Stars & above</label>
+          </div>
+          <div className="filter-item">
+            <input type="radio" name="rating" id="filter-rating" onChange={()=>productDispatch({type:"RATING",payload:3})}/>
+            <label for="">3 Stars & above</label>
+          </div>
+          <div className="filter-item">
+            <input type="radio" name="rating" id="filter-rating" onChange={()=>productDispatch({type:"RATING",payload:2})}/>
+            <label for="">2 Stars & above</label>
+          </div>
+          <div className="filter-item">
+            <input type="radio" name="rating" id="filter-rating" onChange={()=>productDispatch({type:"RATING",payload:1})}/>
+            <label for="">1 Stars & above</label>
+          </div>
+        </div>
+      </aside>
```

Comment:
in rating it is of same name so among all one will be selected at a time so make on function for this from there you call raing sort dispatch